<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JPSeditor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/jupedsim.png</normaloff>:/jupedsim.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_ffnen_cogmap"/>
    <addaction name="action_ffnen_xml"/>
    <addaction name="action_ffnen"/>
    <addaction name="actionSpeichern"/>
    <addaction name="actionSpeichern_dxf"/>
    <addaction name="actionSpeichern_cogmap"/>
    <addaction name="separator"/>
    <addaction name="action_ber"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOnline_Help"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionRoom"/>
    <addaction name="actionAuto_Definition"/>
    <addaction name="actionGather_data"/>
    <addaction name="actionLandmarkWidget"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actiongridmode"/>
    <addaction name="actionShow_Point_of_Origin"/>
    <addaction name="actionRotate_90_deg_clockwise"/>
    <addaction name="separator"/>
    <addaction name="actionanglesnap"/>
    <addaction name="actionObjectsnap"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCopy"/>
    <addaction name="actionScale"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_lines"/>
    <addaction name="actionDelete_single_line"/>
    <addaction name="actionClear_all_Rooms_and_Doors"/>
   </widget>
   <widget class="QMenu" name="menuDraw">
    <property name="title">
     <string>Draw</string>
    </property>
    <addaction name="actionWall"/>
    <addaction name="actionDoor"/>
    <addaction name="actionHLine"/>
    <addaction name="actionLandmark"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDraw"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect_Mode"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionDelete_single_line"/>
   <addaction name="separator"/>
   <addaction name="actionWall"/>
   <addaction name="actionDoor"/>
   <addaction name="actionHLine"/>
   <addaction name="actionLandmark"/>
   <addaction name="separator"/>
   <addaction name="actiongridmode"/>
   <addaction name="actionObjectsnap"/>
   <addaction name="separator"/>
   <addaction name="actionRoom"/>
  </widget>
  <action name="action_ffnen">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/folder-open.svg</normaloff>:/inAppIcons/Icons/folder-open.svg</iconset>
   </property>
   <property name="text">
    <string>Load DXF</string>
   </property>
   <property name="toolTip">
    <string>Load DXF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/save.svg</normaloff>:/inAppIcons/Icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save as XML</string>
   </property>
   <property name="toolTip">
    <string>Save as XML</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="icon">
    <iconset>
     <normaloff>:/man-gimp1.png</normaloff>:/man-gimp1.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_ber">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actiongridmode">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/Grid.png</normaloff>:/inAppIcons/Icons/Grid.png</iconset>
   </property>
   <property name="text">
    <string>Show Grid</string>
   </property>
   <property name="toolTip">
    <string>Gridmode</string>
   </property>
   <property name="shortcut">
    <string>Meta+G</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionWall">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/wall.svg</normaloff>:/inAppIcons/Icons/wall.svg</iconset>
   </property>
   <property name="text">
    <string>Wall</string>
   </property>
   <property name="shortcut">
    <string>Meta+W</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionObjectsnap">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/snaping.png</normaloff>:/inAppIcons/Icons/snaping.png</iconset>
   </property>
   <property name="text">
    <string>Object snap</string>
   </property>
   <property name="toolTip">
    <string>Object snap</string>
   </property>
   <property name="shortcut">
    <string>Meta+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDelete_lines">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/minus-square.svg</normaloff>:/inAppIcons/Icons/minus-square.svg</iconset>
   </property>
   <property name="text">
    <string>Delete all</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDelete_single_line">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/trash-alt.svg</normaloff>:/inAppIcons/Icons/trash-alt.svg</iconset>
   </property>
   <property name="text">
    <string>Delete selected lines</string>
   </property>
   <property name="toolTip">
    <string>Delete selected lines</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDoor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/door-open.svg</normaloff>:/inAppIcons/Icons/door-open.svg</iconset>
   </property>
   <property name="text">
    <string>Door</string>
   </property>
   <property name="toolTip">
    <string>Door</string>
   </property>
   <property name="shortcut">
    <string>Meta+D</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Exit.png</normaloff>:/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/rooms.png</normaloff>:/inAppIcons/Icons/rooms.png</iconset>
   </property>
   <property name="text">
    <string>Define Rooms</string>
   </property>
   <property name="toolTip">
    <string>Define Room</string>
   </property>
   <property name="shortcut">
    <string>Meta+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSelect_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/hand-pointer.svg</normaloff>:/inAppIcons/Icons/hand-pointer.svg</iconset>
   </property>
   <property name="text">
    <string>Select Mode</string>
   </property>
   <property name="toolTip">
    <string>Select Mode</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSpeichern_dxf">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/save.svg</normaloff>:/inAppIcons/Icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save as DXF</string>
   </property>
   <property name="toolTip">
    <string>Save as DXF</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_ffnen_xml">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/folder-open.svg</normaloff>:/inAppIcons/Icons/folder-open.svg</iconset>
   </property>
   <property name="text">
    <string>Load XML</string>
   </property>
   <property name="toolTip">
    <string>Load XML</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRotate_90_deg_clockwise">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/rotate.png</normaloff>:/inAppIcons/Icons/rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate (90 deg clockwise)</string>
   </property>
   <property name="shortcut">
    <string>Meta+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLandmark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/landmark.svg</normaloff>:/inAppIcons/Icons/landmark.svg</iconset>
   </property>
   <property name="text">
    <string>Landmark</string>
   </property>
   <property name="toolTip">
    <string>Landmark</string>
   </property>
   <property name="shortcut">
    <string>Meta+L</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLandmarkWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edit Landmark </string>
   </property>
   <property name="toolTip">
    <string>LandmarkSpecs</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+L</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionanglesnap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/plus.svg</normaloff>:/inAppIcons/Icons/plus.svg</iconset>
   </property>
   <property name="text">
    <string>Orthomode</string>
   </property>
   <property name="toolTip">
    <string>Orthomode</string>
   </property>
   <property name="shortcut">
    <string>Meta+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="action_ffnen_CogMap">
   <property name="icon">
    <iconset>
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>Load CognitiveMap</string>
   </property>
   <property name="toolTip">
    <string>Load CognitiveMap</string>
   </property>
  </action>
  <action name="actionRun_visualisation">
   <property name="text">
    <string>Run visualisation</string>
   </property>
  </action>
  <action name="actionShow_Point_of_Origin">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/flag.svg</normaloff>:/inAppIcons/Icons/flag.svg</iconset>
   </property>
   <property name="text">
    <string>Show Point of Origin</string>
   </property>
   <property name="shortcut">
    <string>Meta+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/undo.svg</normaloff>:/inAppIcons/Icons/undo.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo PaintEvent</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/redo.svg</normaloff>:/inAppIcons/Icons/redo.svg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo PaintEvent</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/hline.png</normaloff>:/inAppIcons/Icons/hline.png</iconset>
   </property>
   <property name="text">
    <string>HLine</string>
   </property>
   <property name="toolTip">
    <string>HLine</string>
   </property>
   <property name="shortcut">
    <string>Meta+H</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSpeichern_cogmap">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/filesave.png</normaloff>:/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save CognitiveMap</string>
   </property>
   <property name="toolTip">
    <string>Save CognitiveMap</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_ffnen_cogmap">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>Load Cognitive Map</string>
   </property>
   <property name="toolTip">
    <string>Load Cognitive Map</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="text">
    <string>Scale</string>
   </property>
  </action>
  <action name="actionAuto_Definition">
   <property name="text">
    <string>Auto Definition</string>
   </property>
  </action>
  <action name="actionGather_data">
   <property name="text">
    <string>Gather Room Type Data</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionClear_all_Rooms_and_Doors">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/inAppIcons/Icons/eraser.svg</normaloff>:/inAppIcons/Icons/eraser.svg</iconset>
   </property>
   <property name="text">
    <string>Clear all Rooms and Doors</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOnline_Help">
   <property name="text">
    <string>Online Help</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
